docs/getting-started/using-gnokey.md

```diff
@@ -0,0 +1,777 @@
+`-broadcast`,`-send`, `-gas-wanted`, `-gas-fee`, `-chain-id`, and `-remote` are
+used for setting the base transaction configuration. These flags will be repeated
+throughout the tutorial.
+
+For this specific demonstration, we will run a local Gno node using `gnodev`.
+First, simply start `gnodev`:
+
+```bash
+gnodev
+```
+
+If everything went well, you should see the following output:
+```bash
+❯ gnodev
+Accounts    ┃ I default address imported name=test1 addr=g1jg8mtutu9khhfwc4nxmuhcpftf0pajdhfvsqf5
+Node        ┃ I pkgs loaded path="[{<your_monorepo_path> g1jg8mtutu9khhfwc4nxmuhcpftf0pajdhfvsqf5 }]"
+Node        ┃ I node started lisn=tcp://127.0.0.1:26657 chainID=dev
+GnoWeb      ┃ I gnoweb started lisn=http://127.0.0.1:8888
+-- READY   ┃ I for commands and help, press `h`
+```
+
+Now we have a local Gno node listening on `127.0.0.1:26657` with chain ID `dev`,
+which we can use to upload our code to.
+
+Next, let's configure the `addpkg` subcommand. Assuming we are in the `example/p`
+folder, the command will look like this:
+
+```bash
+gnokey maketx addpkg \                                                                                                                                                                                          
+-pkgpath "gno.land/p/<your_namespace>/hello_world" \
+-pkgdir "." \
+-send "" \
+-gas-fee 10000000ugnot \
+-gas-wanted 8000000 \
+-broadcast \
+-chainid dev \
+-remote "127.0.0.1:26657" \
+```
+
+Once we have added a desired namespace to upload the package to, we can specify
+a keypair name to use to execute the transaction:
+
+```bash
+gnokey maketx addpkg \                                                                                                                                                                                          
+-pkgpath "gno.land/p/leon/hello_world" \
+-pkgdir "." \
+-send "" \
+-gas-fee 10000000ugnot \
+-gas-wanted 200000 \
+-broadcast \
+-chainid dev \
+-remote "127.0.0.1:26657" \
+mykey
+```
+
+If the transaction was successful, you will get the following output from `gnokey`:
+
+```
+OK!
+GAS WANTED: 200000
+GAS USED:   117564
+HEIGHT:     3990
+EVENTS:     []
+```
+
+Let's analyze the output, which is standard for any `gnokey` transaction:
+- `GAS WANTED: 200000` - the original amount of gas specified for the transaction
+- `GAS USED:   117564` - the gas used to execute the transaction
+- `HEIGHT:     3990` - the block number at which the transaction was executed at
+- `EVENTS:     []` - events emitted by the transaction, in this case, none
+
+Congratulations! You have just uploaded a pure package to your local chain.
+If you wish to upload the package to a remote testnet, make sure to switch out
+the `-chainid` & `-remote` values for the ones matching your desired testnet.
+Find a list of all networks in the [Network Configuration](../reference/network-config.md)
+section.
+
+### `Call`
+
+The `Call` message type is used to call any exported function on the chain.
```

Comment:
Consider rephrasing this as `any exported Realm function...`

Reply:
Rephrased it: [d1bb488](https://github.com/gnolang/gno/pull/2226/commits/d1bb488c3719ebd409e2da08181761161ae10765)